index.md

```diff
@@ -1,8 +1,27 @@
-# Recipes
+# Executable Tutorials
+
+
+## Example recipes
```

Comment:
We switch from "recipes" to "tutorials". Should we say "Example Tutorials" or just "Examples" here?

Reply:
Sure!